Makefile

```diff
@@ -12,77 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CMDS=nfs-provisioner
+all: build
+
+include release-tools/build.make
+
 ifeq ($(REGISTRY),)
 	REGISTRY = quay.io/external_storage/
 endif
+
 ifeq ($(VERSION),)
 	VERSION = latest
 endif
-IMAGE = $(REGISTRY)nfs-ganesha-server-and-provisioner:$(VERSION)
-IMAGE_ARM = $(REGISTRY)nfs-ganesha-server-and-provisioner-arm:$(VERSION)
-MUTABLE_IMAGE = $(REGISTRY)nfs-ganesha-server-and-provisioner:latest
-MUTABLE_IMAGE_ARM = $(REGISTRY)nfs-ganesha-server-and-provisioner-arm:latest
-
 
-all build: build-x86_64
-.PHONY: all build
-
-build-x86_64:
-	GOOS=linux go build ./cmd/nfs-provisioner
-.PHONY: build-x86_64
-
-build-docker build-docker-x86_64:
-	GOOS=linux go build -o deploy/docker/x86_64/nfs-provisioner ./cmd/nfs-provisioner
-.PHONY: build-docker build-docker-x86_64
+IMAGE_ARM = $(REGISTRY)nfs-provisioner-arm:$(VERSION)
+MUTABLE_IMAGE_ARM = $(REGISTRY)nfs-provisioner-arm:latest
 
 build-docker-arm:
 	GOOS=linux GOARCH=arm GOARM=7 go build -o deploy/docker/arm/nfs-provisioner ./cmd/nfs-provisioner
 .PHONY: build-docker-arm
 
-container container-x86_64: build-docker-x86_64 quick-container-x86_64
-.PHONY: container container-x86_64
-
 container-arm: build-docker-arm quick-container-arm
 .PHONY: container-arm
 
-quick-container quick-container-x86_64:
-	docker build -t $(MUTABLE_IMAGE) deploy/docker/x86_64
-	docker tag $(MUTABLE_IMAGE) $(IMAGE)
-.PHONY: quick-container quick-container-x86_64
-
 quick-container-arm:
 	docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 	docker build -t $(MUTABLE_IMAGE_ARM) deploy/docker/arm
 	docker tag $(MUTABLE_IMAGE_ARM) $(IMAGE_ARM)
 .PHONY: quick-container-arm
 
-push-all: push-x86_64 push-arm
-.PHONY: push-all
-
-push push-x86_64: container
-	docker push $(IMAGE)
-	docker push $(MUTABLE_IMAGE)
-.PHONY: push push-x86_64
-
 push-arm: container-arm
 	docker push $(IMAGE_ARM)
 	docker push $(MUTABLE_IMAGE_ARM)
 .PHONY: push-arm
 
-test-all: test test-e2e
-
-test:
-	go test `go list ./... | grep -v 'vendor\|test\|demo'`
-.PHONY: test
-
-test-e2e:
```

Comment:
do we have a replacement for test-e2e?

Reply:
Not yet, this is coming up in a follow-up PR. Currently, these tests are out of date and not being used.